tasks/ldap.yml

```diff
@@ -2,26 +2,26 @@
 # vim: foldmarker=[[[,]]]:foldmethod=marker
 
 - name: Check default ldap config exists
-  command: php --file "{{ owncloud__home }}/occ" ldap:show-config ""
+  command: '{{ owncloud__occ_bin_file_path | quote }} ldap:show-config'
   changed_when: False
   when: (not ansible_check_mode)
   register: owncloud__register_ldap_default_config_exits
   become_user: '{{ owncloud__user }}'
```

Comment:
` become_user` would not be needed when you use the `owncloud__occ_bin_file_path`.  What was the reason for this change anyway? I did not see an urge to change this myself.